omod/src/main/webapp/pages/conditionlist/manageConditions.gsp

```diff
@@ -86,7 +86,7 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
                                ng-click="activateCondition(condition)" ng-if="condition.status==='INACTIVE' && condition.voided===false"></i>
                             <i class="icon-minus-sign edit-action" title="${ui.message("coreapps.conditionui.inactive")}"
                                ng-click="deactivateCondition(condition)" ng-if="condition.status==='ACTIVE' && condition.voided===false"></i>
-                            <i class="icon-remove delete-action" title="${ui.message("coreapps.coreapps.delete")}"
+                            <i class="icon-remove delete-action" title="${ui.message("coreapps.delete")}"
```

Comment:
does coreapps.delete arleady exist in the messages.properties file?

Reply:
Some failing checks here,did this pass locally on building?

Reply:
@HerbertYiga Yes, coreapps.delete already exists in messages.propertis file